src/factory/where.ts

```diff
@@ -0,0 +1,105 @@
+import { SQLTemplate } from '@src/types';
+import { createListOfSqlParams } from '@src/util';
+import { OperatorValue, Queryable, TypeWithCompareOperator } from './types';
+
+export const getFieldQueryPart = <type extends Queryable>(
```

Comment:
I would probably move this function under `wherePart` accordingly to clean code practices. 
In this case I mean the function jump should usually go down, not up. 
`where` calls `wherePart` which in turn calls `getFieldQueryPart` which then calls `mapOperator`